internal_worker.go

```diff
@@ -106,8 +107,11 @@ type (
 		// Defines how many concurrent executions for task list by this worker.
 		ConcurrentActivityExecutionSize int
 
-		// Defines rate limiting on number of activity tasks that can be executed per second.
-		MaxActivityExecutionsPerSecond float32
+		// Defines rate limiting on number of activity tasks that can be executed per MaxActivityExecutionRateRefreshDuration.
+		MaxActivityExecutionRate int
```

Comment:
Since you are anyways renaming this, can you remove *Activity* from  the name. Makes the reader think this applies only to activities, when in fact, this is really the taskExecutionRate